web-platform/website/src/components/pages/account-settings/AccountSettingsPageContent.tsx

```diff
@@ -26,7 +38,17 @@ export function AccountSettingsPageContent() {
               <AccountSettingsNavigationBar />
             </Col>
             <Col>
-              <AccountSettingsContentFactory />
+              <Toast className={`m-auto position-absolute account-toast d-flex justify-content-center ${toastVisible == true ? opacity : ""}`} fade={toastVisible == true} isOpen={toastVisible}>
+                <img
+                  className="align-self-center"
+                  src="img/icon-status-checkmark.svg"
+                  alt="checkmark icon"
+                  width="24px"
```

Comment:
Can we use spacing values here?

width={Spacing.xLarge}
height={Spacing.xLarge}